src/bin/owi.ml

```diff
@@ -42,7 +42,7 @@ let model_format_conv =
     match String.lowercase_ascii s with
     | "scfg" -> Ok Cmd_utils.Scfg
     | "json" -> Ok Json
-    | _ -> Error (`Msg "Expected \"json\" or \"scfg\" but got %s\n")
+    | _ as s -> Fmt.error_msg {|Expected "json" or "scfg" but got "%s"|} s
```

Comment:
Maybe remove the `as s`, because if the users enters `YOLO`, he'll see: `Expected "json" or "scfg" but got yolo` (instead of `YOLO`), which is confusing